cip/1.accepted/CIP2015-08-06-date-time.adoc

```diff
@@ -915,11 +972,11 @@ The temporal types form a separate type system within the cypher type system, th
 
 The previous timestamp function of Cypher still works, and returns the equivalent value of
 
-    datetime().epoch
+    datetime().epochMillis
 
 Perhaps more importantly a value produced by the old +timestamp+ function can be converted to a _DateTime_ by passing it as the `epoch` component to the `datetime` function:
 
-    datetime({epoch:timestamp()})
+    datetime.fromEpochMillis(timestamp())
```

Comment:
Implementation note: We can still support the old way (as a value in a map), but we should also support this new way!

Tobias note: The sentence above does not reflect this changes.

Reply:
I'm don't think we should support the value in a map way for this.

Reply:
After thinking this through, we should go back to the map-based form, but rename the component to `epochMillis`, since this allows specifying a timezone.